PowerShell.sublime-syntax

```diff
@@ -10,9 +10,10 @@ file_extensions:
   - psd1
 
 first_line_match: |
-  (?x)
+  (?x:
```

Comment:
If you use `(?x:` you should also use `|-` for the block indicator to trim the trailing newline.